### PR TITLE
regenerate with gtk 3.99.4

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

File: gsk4/src/auto/render_node.rs [hunk 1]
```diff
@@ -21,7 +21,7 @@ glib_wrapper! {
 }
 
 impl RenderNode {
-    //pub fn deserialize(bytes: &glib::Bytes, error_func: /*Unimplemented*/FnMut(/*Unimplemented*/Fundamental: Pointer, &glib::Error), user_data: /*Unimplemented*/Option<Fundamental: Pointer>) -> Option<RenderNode> {
+    //pub fn deserialize(bytes: &glib::Bytes, error_func: /*Unimplemented*/FnMut(/*Ignored*/gtk::CssSection, &glib::Error), user_data: /*Unimplemented*/Option<Fundamental: Pointer>) -> Option<RenderNode> {
```
Comment (sdroege) [inline gsk4/src/auto/render_node.rs:24]: This causes a circular dependency between gtk4 and gsk4, that's rather suboptimal.

File: gsk4/sys/src/lib.rs [hunk 2]
```diff
@@ -109,7 +109,7 @@ pub const GSK_TRANSFORM_CATEGORY_IDENTITY: GskTransformCategory = 6;
 
 // Callbacks
 pub type GskParseErrorFunc =
-    Option<unsafe extern "C" fn(gconstpointer, *const glib::GError, gpointer)>;
+    Option<unsafe extern "C" fn(*const gtk::GtkCssSection, *const glib::GError, gpointer)>;
```
Comment (sdroege) [inline gsk4/sys/src/lib.rs:112]: Does that compile? :)

**Comment by sdroege** [comment]
Looks good to me otherwise

File: gsk4/sys/src/lib.rs (hunk 2 above)
Comment (bilelmoussaoui) [reply gsk4/sys/src/lib.rs:112]: I don't know for now, waiting on the gtk4 docker image to get updated

**Review by sdroege — APPROVED** [review]

**Pull request merged**